stac_fastapi/types/stac_fastapi/types/core.py

```diff
@@ -305,12 +305,12 @@ def landing_page(self, **kwargs) -> stac_types.LandingPage:
         base_url = str(kwargs["request"].base_url)
         landing_page = self._landing_page(base_url=base_url)
         collections = self.all_collections(request=kwargs["request"])
-        for collection in collections:
+        for collection in collections["collections"]:
             landing_page["links"].append(
                 {
                     "rel": Relations.child.value,
                     "type": MimeTypes.json.value,
-                    "title": collection.get("title"),
+                    "title": collection.get("title") or collection.get("id"),
```

Comment:
Looks like this change is the culprit on the remaining test

Reply:
I should add: this change looks reasonable to me - do we know whether the test encodes something important about the STAC spec?

Reply:
@moradology I don't think the spec has an opinion. At least, not one which I have found. I was working under the assumption it was a standard URL so the title is for display rather than anything else. As you say, seemed reasonable.